resources/backend/api_endpoints.md

```diff
@@ -0,0 +1,35 @@
+# API Endpoints
+API endpoints are essentially points of communication in which a user can interact with and utilize an API's functions. In the context of this project, they are ways for frontend users to interact with and use backend functions.
+![API Endpoint Diagram](../images/api_endpoint_diagram.png)
```

Comment:
The image won't show properly until this gets merged cause the url is different.